packages/cozy-app-publish/lib/hooks/post/mattermost.js

```diff
@@ -7,9 +7,19 @@ const MATTERMOST_ICON =
   'https://travis-ci.com/images/logos/TravisCI-Mascot-1.png'
 const MATTERMOST_USERNAME = 'Travis'
 
-const sendMattermostReleaseMessage = (appSlug, appVersion) => {
-  const message = `__${appSlug}__ version \`${appVersion}\` has been published.`
-
+const sendMattermostReleaseMessage = (
+  appSlug,
+  appVersion,
+  spaceName,
+  appType
```

Comment:
can we default to appType = 'webapp' ? If we don't do it, we'll have an error on `appTypeLabelMap[appType]` if we don't update our current script?

Reply:
If appType is undefined, the label will be ''

```javascript
appTypeLabelMap[appType] || ''
```

And appType is already defaulted to 'webapp' : https://github.com/cozy/cozy-libs/blob/c6b443fd3fa013d209df47ba3ea028d2c7fe7d24/packages/cozy-app-publish/lib/publisher.js#L56

I don't thing we need a default for appType here

Reply:
Woa, didn't see the ` || ''` thanks